packages/esm-ward-app/src/ward-workspace/ward-patient-notes/history/notes-container.test.tsx

```diff
@@ -54,7 +54,7 @@ describe('PatientNotesHistory', () => {
       isLoadingPatientNotes: true,
     });
 
```

Comment:
Minor thing, but `mockPatientNotes` here should have values with `encounterUuid` and  `obsUuid` instead of `id`.